flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/TimeWindowTranslationTest.java

```diff
@@ -51,12 +53,48 @@
  */
 public class TimeWindowTranslationTest {
 
+	@Test
```

Comment:
Well this tests that now even for processing time, the `timeWindow()` leads to a `WindowOperator` being instantiated, rather than an aligned one. I can explicitly
add the `TimeCharacteristic`. What do you think?